Reverting a change introduced in 6de7e7f8b3698 "[drm] upstream sync to v4.10"
(upstream c470abd4fde40ea).

Index: src/drivers/gpu/drm/mgag200/mgag200_ttm.c
===================================================================
--- src.orig/drivers/gpu/drm/mgag200/mgag200_ttm.c	2017-06-20 20:40:45.435756224 +0200
+++ src/drivers/gpu/drm/mgag200/mgag200_ttm.c	2017-06-20 20:40:47.431726640 +0200
@@ -186,6 +186,17 @@
 {
 }
 
+static int mgag200_bo_move(struct ttm_buffer_object *bo,
+			   bool evict, bool interruptible,
+			   bool no_wait_gpu,
+			   struct ttm_mem_reg *new_mem)
+{
+	int r;
+	r = ttm_bo_move_memcpy(bo, evict, no_wait_gpu, new_mem);
+	return r;
+}
+
+
 static void mgag200_ttm_backend_destroy(struct ttm_tt *tt)
 {
 	ttm_tt_fini(tt);
@@ -231,7 +242,7 @@
 	.init_mem_type = mgag200_bo_init_mem_type,
 	.eviction_valuable = ttm_bo_eviction_valuable,
 	.evict_flags = mgag200_bo_evict_flags,
-	.move = NULL,
+	.move = mgag200_bo_move,
 	.verify_access = mgag200_bo_verify_access,
 	.io_mem_reserve = &mgag200_ttm_io_mem_reserve,
 	.io_mem_free = &mgag200_ttm_io_mem_free,
